src/App.jsx

```diff
@@ -1,11 +1,23 @@
-import {Header} from './component'
-import './App.css'
+import { useEffect, useState } from 'react'
+import { Header, Loader } from './component'
+import { Outlet } from 'react-router-dom'
```

Comment:
Un detalle no más, es una buena práctica mantener los imports ordenados en donde vayan todos juntos lo que son de librerías externas y por otro lado los que son de la app. Una buena opción es primero poner los de librerías externas, entonces quedaría: 
```suggestion
import { useEffect, useState } from 'react'
import { Outlet } from 'react-router-dom'

import { Header, Loader } from './component'
```